src/ethereum/ethEngine.ts

```diff
@@ -986,16 +986,16 @@ export class EthereumEngine extends CurrencyEngine<EthereumPlugin> {
       // Otherwise, use the next nonce
       if (
         this.walletLocalData.numUnconfirmedSpendTxs &&
```

Comment:
Missed sketchy null

Reply:
The goal of this PR is not to fix all the possible TS errors but to just not be "worse than Flow". A future commit auto adds the appropriate eslint exception comment which we can remove in future work and slowly clean up.